storefront/i18n/slugs/de.js

```diff
@@ -0,0 +1,21 @@
+export default {
```

Comment:
I think these files are not needed at all and add a lot of weight to the concept, why don't we just define the rewrites in next.js for this case?

https://nextjs.org/docs/api-reference/next.config.js/rewrites

{
        source: '/ueber-uns',
        destination: '/about',
        locale: "de"
      },

a LocalizedUrl wrapper then could just read the config from next.config.js